.github/workflows/main.yml

```diff
@@ -21,11 +21,10 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  matlab-test:
-    name: Run MATLAB Test Script
+  matlab-test-ubuntu:
```

Comment:
As mentioned elsewhere, I believe the Matlab CI runner should use a similar stategy as the one used by the .NET runner (use artifacts from a previous compilation so things can be checked on multiple platforms). 

I am not asking for this to be added here, but it should be very high on the list of priorities for the next PR.